modules/tinymce/src/core/main/ts/init/Render.ts

```diff
@@ -100,19 +99,16 @@ const loadIcons = (scriptLoader: ScriptLoader, editor: Editor, suffix: string) =
   });
 };
 
-const loadPlugins = (editor: Editor, settings: EditorSettings, suffix: string) => {
-  if (Type.isArray(settings.plugins)) {
-    settings.plugins = settings.plugins.join(' ');
-  }
-
-  Tools.each(settings.external_plugins, function (url, name) {
+const loadPlugins = (editor: Editor, suffix: string) => {
+  // These lines should be changed to a getParam and some type of setParam once such an API is available.
```

Comment:
Looks like this comment wasn't cleaned up when you removed the `array` logic? As it definitely is using `getParam` now.

Reply:
No, I kept it for the other instances of editor.settings in this function. It's poorly worded for that though, so I'll reword it.